WorkloadManagementSystem/Utilities/PilotWrapper.py

```diff
@@ -15,21 +14,60 @@
 
 import os
 import tempfile
+import base64
+import bz2
+
+pilotWrapperContent = """#!/bin/bash
+/usr/bin/env python << EOF
+
+# imports
+import os
+import stat
+import tempfile
+import sys
 import shutil
-import tarfile
-import json
 import base64
 import bz2
+import logging
+import time
+import urllib2
+import tarfile
 
-from cStringIO import StringIO
+# setting up the logging
```

Comment:
These scripts in string are quite hard to read and are error prone, wouldn't it be better to put them in separate files ?

Reply:
I understand, but this script (the pilot wrapper) is composed on the fly, it makes not much sense to put it in a separate file.